css/at-rules.json

```diff
@@ -292,6 +292,17 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@keyframes"
   },
+  "@layer": {
+    "syntax": "@layer <layer-name> {\n  <stylesheet>\n}",
+    "interfaces": [
+      "CSSImportRule"
```

Comment:
According to the docs (https://github.com/mdn/data/blob/master/css/at-rules.md#structure-for-simple-at-rules) this property is "are the CSSOM interfaces that belong to the at-rule" I'm not sure what "belong to" means here, but I think there's a good case for not including an `interfaces` property here.

(I don't know anywhere that this property is used in our docs though.)

Reply:
Yes I actually think this is a mistake and should be these two interfaces https://drafts.csswg.org/css-cascade-5/#the-csslayerblockrule-interface which I've changed